src/components/posts/PostDetail.js

```diff
@@ -25,33 +27,33 @@ export default (props) => {
   };
 
   const handleReaction = (postId) => {
-      getPostById(postId).then(setPost)    
-  }
+    getPostById(postId).then(setPost);
+  };
 
   useEffect(() => {
     const postId = parseInt(props.match.params.postId);
     getPostById(postId).then(setPost);
   }, []);
 
   return (
-    <div className="postDetail">
+    <Container>
       <Row>
         <Col xl="8" lg="7" sm="12">
           <h2 className="postDetail__title font-weight-bold">{post.title}</h2>
         </Col>
         <Col xl="4" lg="5" sm="12">
-          {currentUser === post.user.id && (
+          {((currentUser === post.user.id) || (userIsAdmin)) && (
             <Row className="justify-content-end">
               <ConfirmableDeleteButton onDelete={handleDeleteButtonClick} />
               <EditPostButton postId={post.id} />
+              {(localStorage.getItem("is_admin") && post.approved) ? <div> Approved</div> : localStorage.getItem("is_admin") ?  <p>Not Approved</p> : <></>}
```

Comment:
Minor suggestion: This link is kinda long. Considder formatting like this:
{
(localStorage.getItem("is_admin") && post.approved) 
? <div> Approved</div> 
: localStorage.getItem("is_admin") 
?  <p>Not Approved</p> 
: <></>
}